src/main/java/mate/academy/spring/controller/ShoppingCartController.java

```diff
@@ -0,0 +1,46 @@
+package mate.academy.spring.controller;
+
+import mate.academy.spring.mapper.DtoResponseMapper;
+import mate.academy.spring.model.ShoppingCart;
+import mate.academy.spring.model.dto.response.ShoppingCartResponseDto;
+import mate.academy.spring.service.MovieSessionService;
+import mate.academy.spring.service.ShoppingCartService;
+import mate.academy.spring.service.UserService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/shopping-carts")
+public class ShoppingCartController {
+    private final DtoResponseMapper<ShoppingCartResponseDto, ShoppingCart>
+            shoppingCartDtoResponseMapper;
```

Comment:
let's make your mappers names consistent:
```suggestion
    private final DtoResponseMapper<ShoppingCartResponseDto, ShoppingCart>
            shoppingCartResponseMapper;
```